pages/watches/koi.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Casio WSD-F10/F20
+deviceNames: [ koi ]
+label: ayu/koi
+section: install
+layout: aw-install
+installParts: [ install-prepare-adb, install-unlock-adb-round, install-select-method, install-full-adb-push-ext4, install-temp-flash-boot-to-recovery ]
+---
+<div class="callout callout-info">
+    <h4>Model number and variants</h4>
+    <p></p>
+    <p>The "Casio Smart Outdoor Watch WSD-F10" <code>koi</code> and "Casio Pro Trek Smart WSD-F20" <code>ayu</code>share a platform. The only difference is the inclusion of GPS and a modified casing design in the <code>ayu</code> model. Generally, both watches are referred to as <code>koi</code>.
+    </p><p>
+    AsteroidOS has only been tested on <code>ayu</code>, but it should work on <code>koi</code>. The USB access procedure for the WSD-F10 <code>koi</code> is unknown. If you have a WSD-F10 and want to help test, please get in touch with us on <a href="https://matrix.to/#/#Asteroid:matrix.org">#Asteroid:matrix.org</a>.
+    </p><p>
+    The Casio WSD-F21HR and WSD-F30 are completely separate watches and not compatible with <code>koi</code>. Work is currently in progress on the WSD-F21HR.
```

Comment:
the 'hr' in WSD-F21hr should not be capitalised

Reply:
actually, maybe it should. marketing material is not consistent, let's go with all caps `WSD-F21HR`